inlang/source-code/paraglide/paraglide-js/src/cli/commands/compile.test.ts

```diff
@@ -1,7 +1,7 @@
 import { vi, test, expect, beforeEach } from "vitest"
 import memfs from "memfs"
-import mockedFs from "node:fs/promises"
-import fs from "node:fs"
+import mockedFs from "fs/promises"
```

Comment:
should this not be prefixed with node: ?

Reply:
My editor was being goofy & only recognized the types when they did not have the `node:` prefix

Reply:
Fixed it